include/bluetooth/mesh/sensor_srv.h

```diff
@@ -44,20 +44,36 @@ struct bt_mesh_sensor_srv;
  *  @brief Sensor Server model composition data entry.
  *
  *  @param[in] _srv Pointer to a @ref bt_mesh_sensor_srv instance.
+ *  @param ...      Optional Light Lightness Server metadata if application is
```

Comment:
```suggestion
 *  @param ...      Optional Light Lightness server metadata if application is
```

Reply:
Why? Everywhere 'Server' is capitalized. [See](https://github.com/nrfconnect/sdk-zephyr/blob/38452400d3bed75faf30b5a0e4a5a34d9a44128b/include/zephyr/bluetooth/mesh/health_srv.h#L169)